types/src/data_availability_header.rs

```diff
@@ -15,11 +16,17 @@ use crate::{bail_validation, Error, Result, ValidateBasic, ValidationError};
     into = "RawDataAvailabilityHeader"
 )]
 pub struct DataAvailabilityHeader {
-    pub row_roots: Vec<Vec<u8>>,
-    pub column_roots: Vec<Vec<u8>>,
+    pub row_roots: Vec<NamespacedHash>,
+    pub column_roots: Vec<NamespacedHash>,
     pub hash: Hash,
 }
 
+impl DataAvailabilityHeader {
+    pub fn root_hash(&self) -> Option<&NamespacedHash> {
```

Comment:
each row in row_roots are root hashes, in tests we use the 0 index because as only we submit blobs to the blocks, and we can only submit one per block so it always ends up in first row.
We could instead have an api like this
```rs
pub fn row_root(row: usize) -> Option<&NamespacedHash>
pub fn column_root(row: usize) -> Option<&NamespacedHash>
```